src/components/modals/ErrorModal/ErrorModal.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+ErrorModal.propTypes = {
+  handleErrorOK: PropTypes.func.isRequired,
+}
+
+function ErrorModal({ handleErrorOK }) {
+  return (
+    <ErrorWrapper>
+      <Modal>
+        <ErrorTitle>Ein Fehler ist beim Laden aufgetreten!</ErrorTitle>
+        <InfoMessage>
+          Die App wird zurück gesetzt. Alle Daten sind verloren!
+        </InfoMessage>
+        <Action onClick={handleErrorOK}>OK</Action>
+      </Modal>
+    </ErrorWrapper>
+  )
+}
+
+export default ErrorModal
+
+const ErrorWrapper = styled.div`
+  background-color: rgba(0, 0, 0, 0.7);
+  bottom: 0;
+  height: 100vh;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 100wh;
```

Comment:
I think you don't need the width if you have "left:0" and "right:0". You could try removing it.

Reply:
good catch